docs/content/docs.md

```diff
@@ -578,6 +578,21 @@ This flag can be useful for debugging and in exceptional circumstances
 (e.g. Google Drive limiting the total volume of Server Side Copies to
 100GB/day).
 
+### --dscp VALUE ###
+
+Specify a DSCP value or name to use in connections. This could help QoS
+system to identify traffic class. BE, EF, DF, LE, CSx and AFxx are allowed.
+
+See the description of [differentiated services](https://en.wikipedia.org/wiki/Differentiated_services) to get an idea of
+this field. Setting this to 1 (LE) to identify the flow to SCAVENGER class
+can avoid occupying too much bandwidth in a network with DiffServ support ([RFC 8622](https://tools.ietf.org/html/rfc8622)).
+
```

Comment:
@Max-Sum 

Please add https://github.com/rclone/rclone/pull/4266#issuecomment-769554945
to the docs:

```
ECN should be set in OS.
Linux: sysctl net.ipv4.tcp_ecn=1
Windows: netsh interface tcp set global ecncapability=enabled
```

Reply:
ECN is a part of DSCP. I don't think this should be appended.

Reply:
OK